README.md

```diff
@@ -23,6 +23,14 @@ Or install it yourself as:
 $ gem install blueprinter
 ```
 
+## Documentation
+
+We use [Yard](https://yardoc.org/) for documentation. Here are the following
+documentation rules:
+
+- Document public all methods we expect to be utilized by the end developers.
```

Comment:
`nb`: I think you mean `all public` not `public all`

Reply:
nice catch